### PR TITLE
Mark v20230211 as recalled

### PR DESCRIPTION
**Pull request by saurav-agarwalla**
v20230211 is being recalled due to an issue affecting Kernel upgrades. Working on putting up a Github issue explaining the problem in parallel.

**Issue #, if available:**

**Description of changes:**

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Testing Done**

*[See this guide for recommended testing for PRs.](../doc/CONTRIBUTING.md#testing-changes) Some tests may not apply. Completing tests and providing additional validation steps are not required, but it is recommended and may reduce review time and time to merge.*

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-### AMI Release v20230211
+### [RECALLED] AMI Release v20230211
```
Comment (cartermckinnon) [inline CHANGELOG.md:3]: Our previous recalls have used `[Recalled]` (no caps):
```suggestion
### [Recalled] AMI Release v20230211
```

**Review by cartermckinnon — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (saurav-agarwalla) [reply CHANGELOG.md]: Corrected. Seems like the release notes followed a different convention.

**Review by ravisinha0506 — APPROVED** [review]

**Review by Issacwww — APPROVED** [review]

**Review by cartermckinnon — APPROVED** [review]

**Comment by saurav-agarwalla** [comment]
Opened an issue detailing the problem - https://github.com/awslabs/amazon-eks-ami/issues/1193.

**Comment by stevehipwell** [comment]
This recalled version is still showing up as the default version for EKS `v1.25` using the following command.

```shell
aws ssm get-parameter --name /aws/service/eks/optimized-ami/1.25/amazon-linux-2-arm64/recommended --query "Parameter.Value" --output text
```

**Comment by mmerkes** [comment]
Yes, that is a known issue. 1.25 is not publicly available for EKS yet and there is no other AMI to roll back to for 1.25. We'll be rolling out a new AMI this week for all versions, including 1.25.

**Pull request merged**